tools/hlslpm/github.py

```diff
@@ -0,0 +1,144 @@
+from enum import Enum
+from dataclasses import dataclass, field
+import os
+import sys
+from typing import List, Optional
+import requests
+from datetime import datetime, date
+import json
+
+from issue import Category, Issue
+
+
+class GH:
+    def __init__(self):
+        self.accessToken = get_pat()
+
+    def graphql(self, query: str, variables={}):
+        response = requests.post("https://api.github.com/graphql",
+                                 json={'query': query, 'variables': variables},
+                                 headers={'Authorization': f'Bearer {self.accessToken}',
+                                          'Content-Type': 'application/json'})
+
+        response.raise_for_status()
+
+        return response.json()
+
+    def project_items_summary(self):
+        query = read_file("gql/project_item_summary.gql")
+
+        pageInfo = {'endCursor': None, 'hasNextPage': True}
+
+        while pageInfo['hasNextPage']:
+            response = self.graphql(query, {'after': pageInfo['endCursor']})
+
+            items = maybe_get(response, 'data', 'organization',
```

Comment:
`maybe_get` looks like it could return `None`, but the code below assumes it is a dictionary.

Reply:
Yup, but in this case I'd be happy with the error coming from attempting to use the dictionary - this would be a bug in the code rather than an environmental issue.